canonical-websites/www.ubuntu.com
# Robot OS landing page

## Done

* New landing page at `/engage/robot-operating-system-choice`

## TODO

* [x] [Copy doc](https://docs.google.com/document/d/1jvipqvQvTEh8SerFD07m-dGG9N0LMu4g-OW7xwbraO4/)
* [x] Update hero with ~takeover~ **interim** design

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the page looks good and matches the [copy doc](https://docs.google.com/document/d/1jvipqvQvTEh8SerFD07m-dGG9N0LMu4g-OW7xwbraO4/)
- Ensure the form takes you to the right thank you page
